sample/authentication/crypto.go

```diff
@@ -217,7 +217,7 @@ func (au *SGXUSIGAuthenticationScheme) VerifyAuthenticationTag(m []byte, sig []b
 	// UIs. This, for example, can be achieved using some
 	// bootstrapping procedure.
 	epoch, ok := au.epoch[fingerprint]
-	if !ok && ui.Counter == uint64(1) {
+	if !ok {
```

Comment:
The above inline comment seems to mention the assumption about `ui.Counter == uint64(1)`. So does it need some update?

Reply:
Yes, it seems so. I think we should distinguish here "first received UI" from "first generated UI". Is it then as simple as changing "generates its first UI" to "generates its UIs"?

Reply:
Thank you for the update.